airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/streams/report_streams/brands_report.py

```diff
@@ -0,0 +1,137 @@
+        "adGroupName",
+        "adGroupId",
+        "keywordText",
+        "keywordBid",
+        "keywordStatus",
+        "targetId",
+        "searchTermImpressionRank",
+        "targetingExpression",
+        "targetingText",
+        "targetingType",
+        "matchType",
+        "impressions",
+        "clicks",
+        "cost",
+        "attributedDetailPageViewsClicks14d",
+        "attributedSales14d",
+        "attributedSales14dSameSKU",
+        "attributedConversions14d",
+        "attributedConversions14dSameSKU",
+        "attributedOrdersNewToBrand14d",
+        "attributedOrdersNewToBrandPercentage14d",
+        "attributedOrderRateNewToBrand14d",
+        "attributedSalesNewToBrand14d",
+        "attributedSalesNewToBrandPercentage14d",
+        "attributedUnitsOrderedNewToBrand14d",
+        "attributedUnitsOrderedNewToBrandPercentage14d",
+        "unitsSold14d",
+        "dpv14d",
+    ],
+    "adGroups": [
+        "campaignName",
+        "campaignId",
+        "campaignStatus",
+        "campaignBudget",
+        "campaignBudgetType",
+        "adGroupName",
+        "adGroupId",
+        "impressions",
+        "clicks",
+        "cost",
+        "attributedDetailPageViewsClicks14d",
+        "attributedSales14d",
+        "attributedSales14dSameSKU",
+        "attributedConversions14d",
+        "attributedConversions14dSameSKU",
+        "attributedOrdersNewToBrand14d",
+        "attributedOrdersNewToBrandPercentage14d",
+        "attributedOrderRateNewToBrand14d",
+        "attributedSalesNewToBrand14d",
+        "attributedSalesNewToBrandPercentage14d",
+        "attributedUnitsOrderedNewToBrand14d",
+        "attributedUnitsOrderedNewToBrandPercentage14d",
+        "unitsSold14d",
+        "dpv14d",
+    ],
+    "campaigns": [
+        "campaignName",
+        "campaignId",
+        "campaignStatus",
+        "campaignBudget",
+        "campaignBudgetType",
+        "campaignRuleBasedBudget",
+        "applicableBudgetRuleId",
+        "applicableBudgetRuleName",
+        "impressions",
+        "clicks",
+        "cost",
+        "attributedDetailPageViewsClicks14d",
+        "attributedSales14d",
+        "attributedSales14dSameSKU",
+        "attributedConversions14d",
+        "attributedConversions14dSameSKU",
+        "attributedOrdersNewToBrand14d",
+        "attributedOrdersNewToBrandPercentage14d",
+        "attributedOrderRateNewToBrand14d",
+        "attributedSalesNewToBrand14d",
+        "attributedSalesNewToBrandPercentage14d",
+        "attributedUnitsOrderedNewToBrand14d",
+        "attributedUnitsOrderedNewToBrandPercentage14d",
+        "unitsSold14d",
+        "dpv14d",
+    ],
+}
+
+
+class SponsoredBrandsReportStream(ReportStream):
+    """
+    https://advertising.amazon.com/API/docs/en-us/reference/sponsored-brands/2/reports
+    """
+
+    def report_init_endpoint(self, record_type: str) -> str:
+        return f"/v2/hsa/{record_type}/report"
+
+    metrics_map = METRICS_MAP
```

Comment:
Why create a local variable?

Reply:
Its not a local variable, its an abstract property method from ReportStream class.

Reply:
ok, but lets keep the following order:

- props
- init
- methods